WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -57,7 +57,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/wda/pressButton"] respondWithTarget:self action:@selector(handlePressButtonCommand:)],
     [[FBRoute POST:@"/wda/siri/activate"] respondWithTarget:self action:@selector(handleActivateSiri:)],
     [[FBRoute POST:@"/wda/apps/launchUnattached"].withoutSession respondWithTarget:self action:@selector(handleLaunchUnattachedApp:)],
-    [[FBRoute GET:@"/wda/device/info"] respondWithTarget:self action:@selector(handleGetDeviceInfo:)],
```

Comment:
Please keep the previous endpoint

Reply:
is there a reason behind this ?

Reply:
- deletion of the previous endpoint will break existing tests that people have and that use it
- both endpoint can coexist without any issues